package.json

```diff
@@ -197,6 +199,9 @@
 		"*.js": [
 			"wp-scripts lint-js"
 		],
+		"packages/*/**": [
```

Comment:
I need to verify my assumption, but it might not work as intended for PR that has multiple commits. Use case I'm about to test is as follows:
- add `CHANGELOG.md` changes upfront and commit
- add code changes with another commit

Reply:
Yes, this is what actually happened:

> ⚠️  Missing CHANGELOG.md entry for compose package edits.
>    More information: https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs

It works as expected based on the implementation but we need to find a way to improve it :)